e2e/basic_scan_test.go

```diff
@@ -139,9 +139,9 @@ func RunSuccessfulScan(ctx ginkgo.SpecContext, report *ReportFailedConfig, filte
 	ginkgo.By("waiting until scan state changes to done")
 	scanParams = models.GetScansParams{
 		Filter: utils.PointerTo(fmt.Sprintf(
-			"scanConfig/id eq '%s' and state eq '%s'",
+			"scanConfig/id eq '%s' and status/state eq '%s'",
 			*apiScanConfig.Id,
-			models.ScanStateDone,
+			models.AssetScanStatusStateDone,
```

Comment:
I added the reason checks and created an issue to add more e2e tests.

https://github.com/openclarity/vmclarity/issues/1066

I'd like to add more outside of this PR because I think that would delay the merging of this PR a lot.